client/src/modules/patients/registration/registration.html

```diff
@@ -101,7 +92,7 @@
                       <div class="help-block text-right">
                         <p>
                           <a ng-click="PatientRegCtrl.toggleFullDate()" href>
-                            <i class="fa fa-circle-o" ng-class="PatientRegCtrl.dateIndicatorIcon"></i>
+                            <i class="fa fa-calendar-o"></i>
```

Comment:
I think this might change the display @mbayopanda was going for - in one state, it is supposed to be a circle that is filled in and in the other it is supposed to be a circle that is empty.